<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Directorio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="folderLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="browsePushButton">
            <property name="text">
             <string>Examinar</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/browse.png</normaloff>:/browse.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="positionPushButton">
            <property name="text">
             <string>Posición</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/pos.png</normaloff>:/pos.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="listView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Progreso</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="numberOfFilesLabel">
          <property name="text">
           <string>Número de archivos</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Controles</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="playPushButton">
          <property name="text">
           <string>Play</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/play.png</normaloff>:/play.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pausePushButton">
          <property name="text">
           <string>Pause</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/pause.png</normaloff>:/pause.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopPushButton">
          <property name="text">
           <string>Stop</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/stop.png</normaloff>:/stop.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="previousPushButton">
          <property name="text">
           <string>Anterior</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/prev.png</normaloff>:/prev.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextPushButton">
          <property name="text">
           <string>Siguiente</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/next.png</normaloff>:/next.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="volumeLabel">
          <property name="text">
           <string>Volumen: 100%</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="volumeProgressBar">
          <property name="value">
           <number>24</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDial" name="volumeDial"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
